CONTRIBUTING.md

```diff
@@ -20,6 +20,41 @@ Use the [Discussions](https://github.com/github/gh-gei/discussions) tab in this
 
 All work done by the maintainers of this repo is tracked in this repo using Issues. We have a hierarchical backlog with Epics at the top, broken down into Batches then broken down to Tasks (epic/batch/task is indicated via labels on the issues). You can see an example Epic and navigate down from there [here](https://github.com/github/gh-gei/issues/101).
 
+
+## Debugging code 
+
+There are two ways to debug code within VS code. 
+
+### Run GH-GHEI command locally
+Run the following commands in your terminal depending on the provider you are looking to run the commands for. 
+
+Azure DevOps: 
+`dotnet run --project src/ado2gh/ado2gh.csproj -- [command]`
```

Comment:
```suggestion
Azure DevOps: 
- Generic: `dotnet run --project src/ado2gh/ado2gh.csproj -- [command]`
- Example: `dotnet run --project src/ado2gh/ado2gh.csproj -- migrate-repo --ado-org "test-org" --ado-team-project "test-project" --ado-repo "test-ado-repo" --github-org "test-github-org" --github-repo "test-github-repo" `
```
@synthead I liked your idea about adding a real example here! How do you feel about this? Happy to reformat

Reply:
Ooh yeah, I like it!  I would also remove the quotes in the example, since there aren't any special characters in your example.  Or alternatively, you could use a simple arg, like `--help` :slightly_smiling_face: Feel free to format it however it makes sense to you :+1:

Reply:
Updated!